src/pages/dashboard/Dashboard.tsx

```diff
@@ -37,32 +37,23 @@ export function Dashboard() {
 
   useEffect(() => {
     if (!userAddress) return;
-    console.log(userAddress);
+
     api?.query.system
       .account(userAddress)
       .then((data) => {
         // @ts-ignore
         setAccountBalance(JSON.parse(data.data.toString()));
-        console.log(data.toString());
       })
       .catch((e) => console.error(e));
   }, [api, userAddress]);
 
-  const maxBalance = () => {
-    if (Number(miscFrozen) > 0 || Number(feeFrozen) > 0) {
-      return toUnit(utils.nMax(miscFrozen.valueOf(), feeFrozen.valueOf()));
-    }
-
-    return 0;
-  };
-
   return (
     <div class="mt-10">
       <div className="dashboard portfolio">
         <h1>Portfolio</h1>
         <div className="portfolio">
           <h4>Total balance</h4>
-          <h2>{maxBalance()} AMPE</h2>
+          <h2>{toUnit(free) / 1000} AMPE</h2>
```

Comment:
I think I would prefer if this value was wrapped in a `useMemo()` hook to prevent unnecessary re-renders of the component.

Reply:
Agreed with this comment ^ 
Also dividing by 1000 here doesn't seem a good idea (maybe include it in the toUnit()